fastplotlib/graphics/line_collection.py

```diff
@@ -262,6 +276,13 @@ def __init__(
 
         kwargs
             passed to LineCollection
+
+        Examples
+        --------
+        .. code-block::python
+
```

Comment:
simple examples with some sines and cosines, can also just use 1 line each to show that colors & data are indexable just like `LineCollection`